assets/components/src/plugin-settings/SettingsSection.js

```diff
@@ -46,6 +48,8 @@ const getControlType = setting => {
 		case 'boolean':
 		case 'checkbox':
 			return 'checkbox';
+		case 'select':
```

Comment:
This is also not necessary since `SelectControl` is a `<select />` element, which doesn't accept the `type` attribute.

Reply:
Fixed in 4eef95d6